packages/pieces/apps/src/lib/figma/trigger/new-comment.ts

```diff
@@ -0,0 +1,105 @@
+import { createTrigger, TriggerStrategy, Property, assertNotNullOrUndefined } from "@activepieces/framework";
+import { figmaAuth } from '../common/props';
+import { Comment } from '../common/models';
+import { figmaCommon } from "../common";
+import { figmaGetRequest } from '../common/utils';
+
+type TriggerData = {
+  lastCommentCreatedAt: string;
+}
+
+const TRIGGER_DATA_STORE_KEY = 'figma_new_comment_trigger_data'
+
+export const newCommentTrigger = createTrigger({
+  name: 'new_comment',
+  displayName: 'New comment',
+  description: 'Triggers when a new comment is posted',
+  // Update: There's a new Figma API v2 in open beta that supports webhooks!
+  // https://www.figma.com/developers/api#webhooks_v2
```

Comment:
I am considering use cases where a new comment is only useful when it pertains to the entire team rather than a single file. As such, I suggest that we transition to using the webhooks API.

To implement this, we can ask the user for their team ID, register a webhook upon enabling the function, and store the resulting webhook ID. in the run function, we will pass the payload unaltered. on disable function, we will delete the webhook.

What do you think?

Reply:
Yes, agreed. Was going with a code comment only because of the beta status there (their docs say that "features and functions may change"). But I can change it now.

Reply:
webhook version implemented